### DISCUSSION TITLE
Chakra for frameworks other than just React.

### DISCUSSION THREAD

**Discussion by pociej** [chakra-ui/chakra-ui#8210] [General]
Im looking for open source solution that covers both design system and as much as possible framework agnostic components library. Chakra seems really good, has nice theming and great figma kit, but problem I see is that it seems to be really tight coupled with React. 
I saw there is https://github.com/chakra-ui/chakra-ui-vue but, correct me if im wrong, last commit 10 months ago suggest that its kinda deprecated. Is there a way, or plan to have a way to separate concerns so one can easly take css layer and implement components for svelte, solid or whatever?

**Comment by segunadebayo** [comment]
Chakra React stays React. For other frameworks use Ark UI plus Panda CSS.

https://ark-ui.com
